sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Release History
 
+### 12.0.0b5 (Unreleased)
+
+#### Features Added
+- Added support for `max_single_get_size` and `max_chunk_get_size` to `ChangeFeedClient`
```

Comment:
Can you add something like, "to control the initial size and chunk size when downloading the changefeed blobs"? Just to give a bit more context to what those params do since their names aren't super helpful.